### ISSUE TITLE
No posts found after 3.0 update

### ISSUE THREAD

**Issue by MarioKnight** [10up/ElasticPress#1327]
Have you searched for similar issues before submitting this one?
Yes

Is this a bug, question or feature request?
Bug

Describe the issue you encountered:
In the WordPress dashboard, navigating to "All Posts" shows "No posts found" in the body yet the counts for all, mine, etc, are correct.  Going to the blog page yields an error rather than the post listings.

Current WordPress version:
5.2

Current ElasticPress version:
3.0

Current Elasticsearch version:
2.4.6

Where do you host your Elasticsearch server:
Same local server as the website

Other plugins installed (WooCommerce, Simple Redirect Manager, etc..):
WooCommerce, a few others

Steps to reproduce:
Nothing special needed beyond the description above.  Deactivating the plugin will bring back the expected behavior in both the dashboard and front-end blog page.

**Comment by tlovett1** [comment]
What EP features do you have active? Did you try reindexing? I am unable to reproduce.

**Comment by MarioKnight** [comment]
I reindexed after the 3.0 update with no issues.  I reactivated the plugin and the behavior occurred right away, and remained after another reindex.  The EP features that are active are:

* Post Search
* Protected Content
* WooCommerce
* Facets

**Comment by MarioKnight** [comment]
I've kept this plugin disabled, but after 3.0.3 was released, I was hoping that maybe things would get going again.  And I've actually got everything back to working as intended!  A reindex after activating the plugin didn't help, so my plan was to disable a feature and reindex to see if I could determine if it was a feature.  I started with WooCommerce and the posts were still missing, but when I re-enabled the feature which kicked off the auto reindex, the posts came back.  I'm not sure what happened, but I'm glad things are back in working order and will close this ticket.
